### PR TITLE
fix(RHICOMPL-2787) - Configured import-ssg job specific variables

### PR DESCRIPTION
**Pull request by Victoremepunto**
## Secure Coding Practices Checklist GitHub Link
- https://github.com/RedHatInsights/secure-coding-checklist

## Secure Coding Checklist
- [ ] Input Validation
- [ ] Output Encoding
- [ ] Authentication and Password Management
- [ ] Session Management
- [ ] Access Control
- [ ] Cryptographic Practices
- [ ] Error Handling and Logging
- [ ] Data Protection
- [ ] Communication Security
- [ ] System Configuration
- [ ] Database Security
- [ ] File Management
- [ ] Memory Management
- [x] General Coding Practices

### CODE REVIEW

File: clowdapp.yaml [hunk 1]
```diff
@@ -423,6 +423,14 @@ parameters:
   value: 100m
 - name: CPU_REQUEST_FLOO
   value: 50m
+- name: MEMORY_LIMIT_IMPORT_SSG
+  value: 2000Mi
```
Comment (skateman) [inline clowdapp.yaml:427]: It doesn't help, also not needed. Let's just go back to the default.

Comment (Victoremepunto) [reply clowdapp.yaml]: okay ! left it back to 500Mi to 1000Mi

**Review by skateman — APPROVED** [review]
The Seal of Approval

**Pull request merged**